### PR TITLE
Upload 20151025-jackathon-no-5

### PR DESCRIPTION
**Pull request by chaotianyi**

### CODE REVIEW

File: source/_posts/20151025-jackathon-no-5.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+title: "Jackathon #5 -- Read a journalism book in 8 hours"
+author: Initium Lab
```
Comment (andyshuxin) [inline source/_posts/20151025-jackathon-no-5.md]: @chaotianyi You're the author and is entitled to a proper signature.

File: source/_posts/20151025-jackathon-no-5.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+editor: Chao Tianyi
```
Comment (andyshuxin) [inline source/_posts/20151025-jackathon-no-5.md]: @chaotianyi We don't support `editor` field yet... You can just put it in the main text. Thanks

File: source/_posts/20151025-jackathon-no-5.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+date: 2015-10-25
+ 
+tags:
+- journalism
+- lang:English
+- Jackathon
+---
+
```
Comment (andyshuxin) [inline source/_posts/20151025-jackathon-no-5.md]: @chaotianyi  You can put it here...

Comment (chaotianyi) [inline source/_posts/20151025-jackathon-no-5.md]: @andyshuxin Cool.

**Pull request merged**